pkg/virt-launcher/virtwrap/access-credentials/access_credentials_test.go

```diff
@@ -343,12 +339,11 @@ var _ = Describe("AccessCredentials", func() {
 		})
 
 		matched := false
-		mockConn.EXPECT().QemuAgentCommand(cmdSetPassword, domName).MinTimes(1).DoAndReturn(func(funcCmd string, funcDomName string) (string, error) {
-			if funcCmd == cmdSetPassword {
+		mockDomain.EXPECT().SetUserPassword(user, password, libvirt.DomainSetUserPasswordFlags(0)).MinTimes(1).DoAndReturn(
```

Comment:
Should be `Times(1)` exactly?

Reply:
It won't work. The set-user-passowrd behind is called multipe times.

cc @akrejcir

Reply:
We don't need this `DoAndReturn` since the signature of `EXPECT` already does this job.
It was previously needed since we need to check the param from the agent command.
IOW we can drop the `mathced` variable

Reply:
@fossedihelm Done.